res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -62,6 +56,13 @@ var intervalsPerRev = 1200,
     rpm = 33 + 1 / 3,  //Like a real vinyl !!! :)
     alpha = 1.0 / 8,   //Adjust to suit.
     beta = alpha / 32; //Adjust to suit.
+    
+/**************************
+ * Loop Size array 
+ * first 4 values used for Autoloop Not shifted
+ * last 4 values used for Autoloop Shifted
+ **************************/    
+var loopsize = [1, 2, 4, 8, 0.125, 0.25, 0.5, 16];
```

Comment:
DJ Intro (Serato) and VDJ do not have Shift + Auto Loop values; they only have 1, 2, 4, 8. 
I picked 1/8, 1/4, 1/2 and 16  for shift as those were the other values available on the UI for the Shift values. My trial for Serato DJ (the full version) is expired so I can't confirm for that one. But here are the links to the documented mapping by each: 
http://www.virtualdj.com/documents/Numark%20Mixtrack%203%20-%20VirtualDJ%208%20Operation%20Guide.pdf and https://www.numark.com/images/product_downloads/MixtrackPro3-UserGuide-v1.1.pdf